packages/graphql_server/src/test/githubApi.test.ts

```diff
@@ -7,9 +8,34 @@ export async function testGetRepositoryTopics(repoFounder: string, repoName: str
 
 //test function that calls the method and prints out all contributors
 export async function testGetContributorCount(owner: string, repo: string) {
-  console.log(await getContributorCount(owner, repo, ' ')) //here we need to pass an auth token
+  console.log(await getContributorCount(owner, repo, '')) //here we need to pass an auth token
 }
 
+async function contributorCountForTrendingRepos() {
+  const repos = await fetchTrendingRepos('daily')
+  for (let i = 0; i < repos.length - 1; i += 2) {
+    const currentString: string = repos[i]
```

Comment:
change currentString to owner. 
change nextString to name.

Do it for the weekly and monthly as well